src/outputs/getEventData.js

```diff
@@ -17,19 +19,37 @@ const path = `${__dirname}/../../secrets/output-data/event${eventNo}`;
  */
 const outResults = async results => {
     let out = results.map(x => {
-        delete x.event;
-        delete x.pos;
-        delete x.uuid;
+
         let time = moment.duration(x.time);
-        x.time = moment({minutes: time.minutes(), seconds: time.seconds()}).format('mm:ss');
-        delete x.firstEvent;
-        delete x.pb;
+        x.time = moment({ hours: time.hours(), minutes: time.minutes(), seconds: time.seconds()}).format('HH:mm:ss');
```

Comment:
surely this can be just `x.time = time.format('HH:mm:ss');` ?

Reply:
Time is a duration and durations don't have a method `format` so it has to be changed into a moment first